### DISCUSSION TITLE
Cat

### DISCUSSION THREAD

**Discussion by watameln** [Epix-Incorporated/Adonis#1236] [General]
![image](https://github.com/Epix-Incorporated/Adonis/assets/90981615/a7e529b7-f416-4662-b843-b3e71f12a895)

lets chat about cats

**Comment by Dimenpsyonal** [comment]
Gato

**Reply by moo1210** [reply]
same
![chrome_kWhxjYj88x](https://github.com/Epix-Incorporated/Adonis/assets/27241938/3652b703-0b5d-41ce-bf5c-7df876558e5c)

**Comment by hallowxd** [comment]
My cat is the cutest cat and you can't deny it
